frontend/index.js

```diff
@@ -87,7 +87,7 @@ function makeChange(doc, context, options) {
     seq: state.seq,
     startOp: state.maxOp + 1,
     deps: state.deps,
-    time: (options && typeof options.time === 'number') ? options.time :
+    time: (options && typeof options.time === 'number') ? options.time : // eslint-disable-line
```

Comment:
There is some variation about whether the colon appears on the following line or here. It looks like usually it's a colon on the next line.